### PR TITLE
Update GenericJMX docs with more descriptive fields

### PR DESCRIPTION
**Pull request by mstumpfx**
descriptions taken from https://collectd.org/wiki/index.php/Plugin:GenericJMX

### CODE REVIEW

File: docs/monitors/collectd-activemq.md [hunk 1]
```diff
@@ -23,9 +23,9 @@ Monitor Type: `collectd/activemq`
 | `name` | no | `string` |  |
 | `serviceName` | no | `string` | This is how the service type is identified in the SignalFx UI so that you can get built-in content for it.  For custom JMX integrations, it can be set to whatever you like and metrics will get the special property `sf_hostHasService` set to this value. |
 | `serviceURL` | no | `string` | The JMX connection string.  This is rendered as a Go template and has access to the other values in this config. NOTE: under normal circumstances it is not advised to set this string directly - setting the host and port as specified above is preferred. (**default:** `service:jmx:rmi:///jndi/rmi://{{.Host}}:{{.Port}}/jmxrmi`) |
-| `instancePrefix` | no | `string` |  |
-| `username` | no | `string` |  |
-| `password` | no | `string` |  |
+| `instancePrefix` | no | `string` | Prefixes the generated plugin instance with prefix. If a second InstancePrefix is specified in a referenced MBean block, the prefix specified in the Connection block will appear at the beginning of the plugin instance, the prefix specified in the MBean block will be appended to it |
```
Comment (keitwb) [inline docs/monitors/collectd-activemq.md]: Make the option names start with lower case since that is what users will see.

Comment (keitwb) [reply docs/monitors/collectd-activemq.md]: Also put them in backquotes.

Comment (mstumpfx) [reply docs/monitors/collectd-activemq.md]: added

File: docs/monitors/collectd-activemq.md [hunk 2]
```diff
@@ -23,9 +23,9 @@ Monitor Type: `collectd/activemq`
 | `name` | no | `string` |  |
 | `serviceName` | no | `string` | This is how the service type is identified in the SignalFx UI so that you can get built-in content for it.  For custom JMX integrations, it can be set to whatever you like and metrics will get the special property `sf_hostHasService` set to this value. |
 | `serviceURL` | no | `string` | The JMX connection string.  This is rendered as a Go template and has access to the other values in this config. NOTE: under normal circumstances it is not advised to set this string directly - setting the host and port as specified above is preferred. (**default:** `service:jmx:rmi:///jndi/rmi://{{.Host}}:{{.Port}}/jmxrmi`) |
-| `instancePrefix` | no | `string` |  |
-| `username` | no | `string` |  |
-| `password` | no | `string` |  |
+| `instancePrefix` | no | `string` | Prefixes the generated plugin instance with prefix. If a second `instancePrefix` is specified in a referenced MBean block, the prefix specified in the Connection block will appear at the beginning of the plugin instance, the prefix specified in the MBean block will be appended to it |
+| `username` | no | `string` | Use name to authenticate to the server |
```
Comment (keitwb) [inline docs/monitors/collectd-activemq.md]: shouldn't this be like `username to authenticate with to the server`?

File: docs/monitors/collectd-activemq.md [hunk 3]
```diff
@@ -23,9 +23,9 @@ Monitor Type: `collectd/activemq`
...
+| `password` | no | `string` | Use password to authenticate to the server |
```
Comment (keitwb) [inline docs/monitors/collectd-activemq.md]: Same thing.

File: docs/monitors/collectd-activemq.md (hunk 2 above)
Comment (mstumpfx) [reply docs/monitors/collectd-activemq.md]: copy pasted from their own docs 🙈 will update ¯\_(ツ)_/¯

File: docs/monitors/collectd-activemq.md (hunk 3 above)
Comment (mstumpfx) [reply docs/monitors/collectd-activemq.md]: fixed

**Pull request merged**